docs/scripts/api/buildComponentsDocumentation.ts

```diff
@@ -329,24 +336,51 @@ const buildComponentDocumentation = async (options: {
         return [] as any;
       }
 
-      let description = generatePropDescription(prop, propName);
-      description = renderMarkdownInline(description);
-
+      const {
+        deprecated,
+        jsDocText,
+        signature: signatureType,
+        signatureArgs,
+        signatureReturn,
+        requiresRef,
+      } = generatePropDescription(prop, propName);
+      let description = renderMarkdownInline(jsDocText);
+
+      const additionalPropsInfo: { sx?: boolean; classes?: boolean } = {};
       if (propName === 'classes') {
-        description += ' See <a href="#css">CSS API</a> below for more details.';
+        additionalPropsInfo.classes = true;
+        // description += ' See <a href="#css">CSS API</a> below for more details.';
       } else if (propName === 'sx') {
-        description +=
-          ' See the <a href="/system/getting-started/the-sx-prop/">`sx` page</a> for more details.';
+        additionalPropsInfo.sx = true;
+        // description +=
+        //   ' See the <a href="/system/getting-started/the-sx-prop/">`sx` page</a> for more details.';
```

Comment:
Would be great to keep this link to the sx prop docs here!

Reply:
Should be done [here](https://github.com/mui/material-ui/blob/e35f9ae7694beef810c04f36368a936b630d8f12/docs/src/modules/components/PropertiesTable.js#L87-L97), but the same problem occurs on core.

It's a naming issue. `additionalPropsInfo` should be replace by `additionalInfo` in the script and run `yarn docs:api` (or the opposite)

It will remain a small design issue: the parent `<p/>` has a display flew that should be removed, otherwise the styling is very special 😅

![image](https://github.com/mui/mui-x/assets/45398769/0e5defd0-afb9-4355-be5a-3f0faa57fb0c)

Reply:
Alright, I've updated the typing a bit, so that TypeScript won't allow for a typo like this:

<img width="1121" src="https://github.com/mui/mui-x/assets/13808724/f88fea01-5c70-4fc2-b574-8d391175ac00">

Reply:
I've submitted a PR in the core repo to fix it https://github.com/mui/material-ui/pull/38183